src/app/public/modules/popover/popover-content.component.html

```diff
@@ -12,20 +12,36 @@
   #popoverRef
 >
   <div
-    class="sky-popover sky-shadow sky-rounded-corners"
+    class="sky-popover sky-shadow"
```

Comment:
You can probably only use the `sky-shadow` for default theme, as you're overriding its properties with the `elevation` classes in modern theme.

Reply:
Updated